test/unit/errors/error-collector.test.js

```diff
@@ -2370,3 +2370,76 @@ test('When using the async listener', (t) => {
     })
   }
 })
+
+tap.test('_processErrors', (t) => {
+  let transaction = null
+  let errorCollector = null
+
+  t.beforeEach(() => {
```

Comment:
```suggestion
  t.beforeEach((t) => {
```

It gets a little muddy with the multiple shadows of `t`, but:

```js
tap.test('a test', t => { // this t is the test object for this outer test
	// t.{beforeEach, test, end} all reference the above t
	t.beforeEach(t => { // this t is a reference to the subtest being executed
		t.context.foo = 'foo'
	})

	t.test('subtest', async t => { // this t is local to the subtest (and was augmented by beforeEach)
		const { foo } = t.context
		t.equal(foo, 'foo')
	})

	t.end() // marks that the outer test has finished
})
```